app/org/sagebionetworks/bridge/models/accounts/ParticipantOptions.java

```diff
@@ -0,0 +1,61 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import javax.annotation.Nullable;
+
+import org.sagebionetworks.bridge.dao.ParticipantOption.SharingScope;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Object to deserialize JSON update of participant options. Some or all of the 
+ * values can be missing in the JSON and we will only update values that are non-null.
+ */
+public final class ParticipantOptions {
+
+    private final @Nullable SharingScope sharingScope;
+    private final @Nullable Boolean notifyByEmail;
+    private final @Nullable String externalId;
+    private final @Nullable Set<String> dataGroups;
+    private final @Nullable LinkedHashSet<String> languages;
```

Comment:
As is, if we add a new Participant Option, we'll need to update this class, the DAO, and every test that uses the ParticipantOptions constructor. This means that adding a new Participant Option is quadratically difficult with a rather large coefficient.

I feel like instead of top-level attributes, we should just have a string-to-string map. This way, we only need to update the enum and the direct consumer of the option.